lib/pavilion/plugins/commands/clean.py

```diff
@@ -215,10 +221,12 @@ def get_month_delta(months):
     cur_year = today.year
     cur_day = today.day
     cur_month = today.month
-    cur_time = today.time
+    cur_time = today.time().hour
 
     if cur_month - months <= 0:
```

Comment:
All of this should be done with the datetime timedelta object.

Need now - 12 months?
```python
import datetime as dt
months = 12
dt.datetime.now() - dt.timedelta(days=30*months)
```

Reply:
30 days is a reasonable assumption for the length of a month. It's almost right 100% the time, depending on your definition of almost.

Reply:
deleted this whole function because with the new implementation, it only needs ~2 lines